docs/securing_fledge.rst

```diff
@@ -0,0 +1,202 @@
+.. |enable_https| image:: images/enable_https.jpg
+.. |connection_https| image:: images/connection_https.jpg
+.. |auth_options| image:: images/authentication.jpg
+.. |login| image:: images/login.jpg
+.. |login_dashboard| image:: images/login_dashboard.jpg
+.. |user_pulldown| image:: images/user_pulldown.jpg
+.. |profile| image:: images/profile.jpg
+.. |password| image:: images/password.jpg
+.. |user_management| image:: images/user_management.jpg
+.. |add_user| image:: images/add_user.jpg
+.. |change_role| image:: images/change_role.jpg
+.. |reset_password| image:: images/reset_password.jpg
+.. |certificate_store| image:: images/certificate_store.jpg
+.. |update_certificate| image:: images/update_certificate.jpg
+
+.. Links
+.. |filter_plugins| raw:: html
+
+   <a href="fledge_plugins.html#filer-plugins">Filter Plugins</a>
+
+
+***************
+Securing Fledge
+***************
+
+The default installation of a Fledge service comes with security features turned off, there are several things that can be done to add security to Fledge.  The REST API by default support unencrypted HTTP requests, it can be switched to require HTTPS to be used. The REST API and the GUI can be protected by requiring authentication to prevent users being able to change the configuration of the Fledge system. Authentication can be via username and password or by means of an authentication certificate.
+
+Enabling HTTPS Encryption
+=========================
+
+Fledge can support both HTTP and HTTPS as the transport for the REST API used for management, to switch between there two transport protocols select the *Configuration* option from the left-hand menu and the select *Admin API* from the configuration tree that appears,
+
++-------------+
+| |admin_api| |
++-------------+
+
+The first option you will see is a tick box labeled *Enable HTTP*, to select HTTPS as the protocol to use this tick box should be deselected.
+
++----------------+
+| |enable_https| |
++----------------+
+
+When this is unticked two options become active on the page, *HTTPS Port* and *Certificate Name*. The HTTPS Port is the port that Fledge will listen on for HTTPS requests, the default for this is port 1995.
+
+The *Certificate Name* is the name of the certificate that will be used for encryption. The default s to use a self signed certificate called *fledge* that is created as part of the installation process. This certificate is unique per fledge installation but is not signed by a certificate authority. If you require the extra security of using a signed certificate you may use the Fledge :ref:`certificate_store` functionality to upload a certificate that has been created and signed by a certificate authority.
+
+After enabling HTTPS and selecting save you must restart Fledge in order for the change to take effect. You must also update the connection setting in the GUI to use the HTTPS transport and the correct port.
+
++--------------------+
+| |connection_deselected| |
++--------------------+
```

Comment:
Unknown target name '|connection_deselected|' or you have missed screenshot to add, if yes then fix Malformed table format?